mne/viz/_brain/_timeviewer.py

```diff
@@ -854,7 +849,7 @@ def configure_point_picking(self):
 
     def load_icons(self):
         from PyQt5.QtGui import QIcon
-        resources.qInitResources()
+        from ...icons import resources  # noqa
```

Comment:
rather than a `noqa` and magic init-on-import, I'd leave it closer to how it was and do
```
from ...icons import resources
resources.qInitResources()
```
BTW any problem with calling this multiple times (which will happen if you load multiple windows)? If so then a little helper `init_resources` that inits them exactly once could be helpful.

Reply:
No problem at all, I tested it with `link_brains()`.